<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>362</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>362</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>362</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #404040;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>601</width>
     <height>362</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>601</width>
     <height>362</height>
    </size>
   </property>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mult">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 10px;
padding: 5px;
margin: 4px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="credit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cl_b">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="op_b">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
border-radius: 4px;
margin: 5px;
padding: 10px;

background-color: #232323;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="graph">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;
}

QPushButton:pressed {
	background-color:#A9A9A9;
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_val">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #232323;
border: 1px solid grey;
border-radius: 4px;
padding: 5px;
margin: 5px;</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
